sdk/testing/src/test/java/io/opentelemetry/sdk/testing/assertj/AttributeAssertionTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.assertj;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import io.opentelemetry.api.common.AttributeKey;
+import java.util.function.Consumer;
+import org.assertj.core.api.AbstractAssert;
+import org.junit.jupiter.api.Test;
+
+class AttributeAssertionTest {
+
+  @Test
+  void nullAttr_errorMessageContainsAttrName() {
+    AttributeKey<String> key = AttributeKey.stringKey("flib");
+    AttributeAssertion attrAssertion = AttributeAssertion.create(key, AbstractAssert::isNotNull);
+    AbstractAssert<?, ?> anAssert = AttributeAssertion.attributeValueAssertion(key, null);
+    Consumer<AbstractAssert<?, ?>> a = attrAssertion.getAssertion();
+    String message = assertThrows(AssertionError.class, () -> a.accept(anAssert)).getMessage();
+    assertThat(message).isEqualTo("[STRING attribute 'flib'] \nExpecting actual not to be null");
```

Comment:
your mark is forever on the project, now!

Reply:
Yeah, sorry not sorry. 🙃